README.md

```diff
@@ -23,13 +23,19 @@ instead of the System WebView. Requires cordova-android 4.0 or greater.
 
 The following directions are for cordova-cli (most people).  Alternatively you can use the [Android platform scripts workflow](PlatformScriptsWorkflow.md).
 
-* Open an existing cordova project, with cordova-android 4.0.0+, and using the latest CLI (If below Cordova 5.2.0 which support default variable, please append those values like `--variable XWALK_VERSION="14+" --variable XWALK_COMMANDLINE="--disable-pull-to-refresh-effect" --variable XWALK_MODE="embedded"`).
+* Open an existing cordova project, with cordova-android 4.0.0+, and using the latest CLI. Crosswalk variables can be configurable as a soft option when installing the plugin
 * Add this plugin
 
 ```
 $ cordova plugin add cordova-plugin-crosswalk-webview
 ```
 
+If the version of CLI is below Cordova 5.2.0, these default variables need be appended at install time
```

Comment:
If Cordova CLI version is lower than 5.2.0, these default variables still need to be specified when installing the plugin.
If Cordova CLI version is upper than 5.2.0, they shouldn't need to be specified.